java-spring-security/src/main/resources/static/index.html

```diff
@@ -32,32 +60,37 @@
       fetch('/high_latency');
     }
 
-    function fetchCustomMetric() {
-      fetch('/custom_metric');
+    function fetchCustomMetric(direction) {
+      if (direction) 
+        fetch('/custom_metric?inc=1');
+      else
+        fetch('/custom_metric');
     }
-
   </script>
 </head>
 <body>
-<div>
+<div class="container">
+  <div class="header">
+    <h1>Custom Application Metrics</h1>
+    <p>Click the buttons below to update custom metrics</p>
+  </div>
   <ul>
     <li>
-      <button type="button" onclick="fetchSimple()">SIMPLE</button>
+      <button type="button" onclick="fetchSimple()">Increment Simple counter</button>
     </li>
 
     <li>
-      <button type="button" onclick="fetchHighLatency()">HIGH LATENCY</button>
+      <button type="button" onclick="fetchHighLatency()">Call an endpoint with high latency</button>
     </li>
 
     <li>
-      <button type="button" onclick="fetchCustomMetric()">CUSTOM METRIC</button>
+      <button type="button" onclick="fetchCustomMetric(true)">Increment Custom gauge</button>
+      <button type="button" onclick="fetchCustomMetric()">Decrement Custom gauge</button>
     </li>
 
-    <li/>
-
     <li>
-      <a href="/actuator/prometheus">
-        <button type="button">SEE THE METRICS</button>
+      <a href="/actuator/prometheus" target="_blank">
```

Comment:
I tend to not like automatic new windows, but if you think this is less confusing I'm ok with it.

Reply:
Yeah I thought it was less confusing since it kept your place in the app.